src/test_files/googleFormAppScript

```diff
@@ -0,0 +1,29 @@
+var POST_URL = "http://02b47c3814ce.ngrok.io";
+
+function onSubmit(e) {
+  var form = FormApp.getActiveForm();
+  var formResponses = form.getResponses();
+  var formLastResponse = formResponses[formResponses.length - 1];
+  var formLastResponseItems = formLastResponse.getItemResponses();
+  var questionsAndAnswers = [];
+  for(var i = 0; i < formLastResponseItems.length; i++) {
+    var question = formLastResponseItems[i].getItem().getTitle();
+    var answer = formLastResponseItems[i].getResponse();
+    questionsAndAnswers.push({
+      "question": question,
+      "answer"  : answer
+    })
+  }
+  
+  UrlFetchApp.fetch(POST_URL, {
+    "method": "post",
+    "contentType": "application/json",
+    "payload": JSON.stringify({
+      "content": questionsAndAnswers
+    })
+  })
+};
```

Comment:
it's kind hard to decide whether this should be part of our library, I know it's a UI component for the library, but imagine the develop do the npm install to install our node library, they also need to deploy this code to google form. It just kinda wired.

Reply:
is not part of the library. basically everything i've put in test_files is just for the purpose of testing this is just something simple to test out things. will get removed in the end since it wont be needed.